docs/dev_install.rst

```diff
@@ -1,20 +1,38 @@
 Installation for development
 ============================
 
-.. code-block:: bash
+#. Clone repo and install requirements:
 
-    $ sudo apt-get install virtualenvwrapper
-    $ mkvirtualenv mmpy_bot
-    $ git clone https://github.com/attzonko/mmpy_bot.git
-    $ cd mmpy_bot
-    $ python setup.py develop
-    $ pip install -r requirements.txt
-    $ pip install -r docs/requirements.txt
-    $ touch local_settings.py               # configure your local settings
-    $ mmpy_bot                              # run bot
+    .. code-block:: bash
 
+        $ git clone https://github.com/attzonko/mmpy_bot.git
+        $ cd mmpy_bot
+        $ pip install -r requirements.txt
+        $ pip install -r dev-requirements.txt
```

Comment:
Why don't we provide the same command used by the workflow `pip install -e  .[dev]`